### PR TITLE
create new contract tabs for smart contracts

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

- Add a new top tab "Contracts" to separate the Smart Contracts documentation from the LSP Standard documentations.

image

This new tab will contain the following docs:
- Introduction
- Overview of each smart contracts 
- Technical references for each contracts (ABI + description of each methods)

image

- add the first pages `introduction.md` and `interfaceIds.md`, migrating the content from the Standards/Smart-Contracts pages into these new pages.
- add http redirect in docusaurus config from the old pages to the new pages.
- keep the pages in the old section, but add a notice with a redirection link
- add the interface ID table as an mdx file `_interface_ids_table.mdx_`, that is autogenerated in the [lukso-network/lsp-smart-contracts](https://github.com/lukso-network/lsp-smart-contracts) Github repository and will be checked automatically for changes

image

### CODE REVIEW

**Review by b00ste — APPROVED** [review]

File: docs/standards/smart-contracts/introduction.md [hunk 1]
```diff
@@ -5,95 +5,6 @@ sidebar_position: 1
 
 # Smart Contract Implementation
 
-:::success Documentation
-
-The smart contracts are public and open source. They can be found [on GitHub](https://github.com/lukso-network/lsp-smart-contracts) and in the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) NPM package.
-
+:::info
+This page has been moved to a new section: [**Contracts -> Introductions**](../../contracts/introduction.md)
```
Comment (Hugoo) [inline docs/standards/smart-contracts/introduction.md:9]: You have set a redirect, you can simply delete this page.

Delete all old pages and use redirects from the config file as much as possible.

don't do both

File: docusaurus.config.js [hunk 2]
```diff
@@ -85,6 +85,15 @@ module.exports = {
...
             to: '/guides/key-manager/execute-relay-transactions',
           },
+          // Smart Contracts new structure
+          {
+            from: '/standards/smart-contracts/introduction',
+            to: '/contracts/introduction',
+          },
+          {
+            to: '/contracts/interface-ids',
+            from: '/standards/smart-contracts/interface-ids',
```
Comment (Hugoo) [inline docusaurus.config.js:95]:
```suggestion
            from: '/standards/smart-contracts/interface-ids',
            to: '/contracts/interface-ids',
```

just for consistency, makes tihngs easier to read as they are all in the same order

File: sidebars.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ module.exports = {
   networksSidebar: [{ type: 'autogenerated', dirName: 'networks' }],
   standardsSidebar: [{ type: 'autogenerated', dirName: 'standards' }],
   guidesSidebar: [{ type: 'autogenerated', dirName: 'guides' }],
+  contractsSidebar: ['contracts/introduction', 'contracts/interface-ids'],
```
[the remark this answers is not in the document]
Comment (CJ42) [reply sidebars.js:16]: that's a good point

[the remark this answers is not in the document]
Comment (CJ42) [reply sidebars.js:16]: Oh yes I know why. The reason is I want to then add a divider between the first two item in the sidebar and the next section under the folder `overview/`

https://github.com/lukso-network/docs/pull/546/files#diff-9a6a0fdafefb0eecffb77784bc94f5ad984062167b8bcb7c4a899c90f6f567e8

Like this: 
image

**Pull request merged**